yt/visualization/base_plot_types.py

```diff
@@ -220,10 +223,16 @@ def _init_image(self, data, cbnorm, cblinthresh, cmap, extent, aspect):
             cmap = get_brewer_cmap(cmap)
         vmin = float(self.zmin) if self.zmax is not None else None
         vmax = float(self.zmax) if self.zmax is not None else None
-        self.image = self.axes.imshow(
-            data.to_ndarray(), origin='lower', extent=extent, norm=norm,
-            vmin=vmin, vmax=vmax, aspect=aspect, cmap=cmap,
-            interpolation='nearest')
+        if self._transform:
```

Comment:
I found that adding in a `transform=None` argument to imshow resulted in all of the non-transformed figures to be blank. Not passing it keeps all of the non-transformed figures behaving as normal in yt, so I needed to add this logic to ensure that no core plotting to yt changes with this PR. I've also filed an issue with matplotlib about passing a None kwarg here: https://github.com/matplotlib/matplotlib/issues/12551

Reply:
the `.transform` attribute doesn't seem to exist by default. But suppose this may be `None`, you may replace it by `ax.transData`.

Reply:
Thanks @ImportanceOfBeingErnest ! I updated the function to set it as `ax.transData`.